### DISCUSSION TITLE
404 Error, HTML file not found

### DISCUSSION THREAD

**Discussion by jelders25** [community/community#60646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

link to repo: https://github.com/jelders25/jackstore.us
custom domain: jackstore.us 

I have verified that everything is setup correctly with the DNS. I have also made sure that all of my HTML files are lower case and not in a folder. Not sure what to do here. Can anyone help?

**Comment by yoannchaudet** [comment]
Rename your `home.html` file to `index.html`. This way you will be able to access it at the URL you expect.

Now it's there: https://www.jackstore.us/home.html

**Reply by jelders25** [reply]
Worked! Thank you

**Reply by queenofcorgis** [reply]
@jelders25 don't forget to mark Yoann's reply as the answer for any other members who may have the same question. Thank you!
